app/views/articles/show.html.erb

```diff
@@ -30,13 +30,16 @@
       <%= heading_tag article.title %>
 
       <% unless @newsletter_excluded %>
-        <%= render 'shared/news_signup_test',
-                   ab_test_render: true %>
+        <%= render 'shared/news_signup_test', ab_test_render: true %>
         <%= render 'shared/news_signup_inpage', ab_test_render: false %>
```

Comment:
Do we still need the _test_ version of this partial now that the A/B test is over?